infrastructure/main.tf

```diff
@@ -74,3 +84,17 @@ module "ccd-admin-web" {
     ADMINWEB_STATE_URL = "${local.def_store_url}/api/data/case-type/{id}"
   }
 }
+
+// Copy into Azure Key Vault
+
+resource "azurerm_key_vault_secret" "idam_service_key" {
+  name = "idam-service-key"
```

Comment:
Conflicts with `ccd-api-gateway/idam-service-key` and `ccd-case-print-service/idam-service-key`